bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4254,6 +4254,40 @@
             INSERT INTO role_role(child_role, parent_role) VALUES ('Appointments:FullAccess', 'Appointments:ManageAppointments');
         </sql>
     </changeSet>
+    <changeSet id="201802211437" author="ningosi">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT COUNT(*) FROM global_property where property = 'patientIdentifierSearch.matchMode';
+            </sqlCheck>
+        </preConditions>
+        <comment>Specify how patient identifiers are matched while searching for a patient</comment>
+        <sql>
+            UPDATE  global_property SET property_value = 'ANYWHERE' WHERE property = 'patientIdentifierSearch.matchMode';
+        </sql>
+    </changeSet>
+
+    <changeSet id="201802281408" author="ningosi">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT COUNT(*) FROM global_property where property = 'patientSearch.matchMode';
+            </sqlCheck>
+        </preConditions>
+        <comment>Specify how patient names are matched while searching for a patient</comment>
```

Comment:
Same as above comment. Say why we're doing this, e.g. changing away from some default? (I thought this was already the default name search in OpenMRS)